### ISSUE TITLE
azcopy list should not modify job files

### ISSUE THREAD

**Issue by cuwaters** [Azure/azure-storage-azcopy#1181]
### Which version of the AzCopy was used? 
10.6.0
### What command did you run? 
azcopy list

### What problem was encountered?
All of the files in ~/.azcopy were modified as part of the list command, making it impossible to determine the job date by the file modification date

### How can we reproduce the problem in the simplest way?
run azcopy list

### Have you found a mitigation/solution?
no

I ran a bunch of really large azcopy jobs.  Then I wanted to go back and get the summary for each job.  I ran azcopy list, then realized I could (I thought) just sort by LastModifiedDate in c:\users\<username>\.azcopy, and pick the jobs with the latest dates.  Unfortunately, running azcopy list touches each file, making the LastModifiedDate the same for all of the jobs in the folder.

**Comment by adreed-msft** [comment]
To clarify: This is a known issue with `azcopy jobs list`, but not one with `azcopy list`. Is this a typo?

**Comment by cuwaters** [comment]
> To clarify: This is a known issue with `azcopy jobs list`, but not one with `azcopy list`. Is this a typo?

@adreed-msft yes, sorry.  I mean azcopy jobs list

**Comment by zezha-msft** [comment]
Hi @cuwaters, the result of `azcopy jobs list` is already sorted by the start date, does that solve your scenario?

**Comment by zezha-msft** [comment]
It's indeed surprising that the files were modified. We should investigate.

**Comment by tasherif-msft** [comment]
Hi @cuwaters, sorry for the late response, I attempted to repro this using Windows with the following steps:

1. Ran a simple job (copy from storage to local dir) 
2. Validated log file for the job was created and its lastModTime
3. Ran `azcopy jobs list` 
4. Checked `.log` file again for step 2 - the lastModTime remained the same, i.e, the job list didn't impact it

is there something I'm missing? are you still experiencing this? otherwise I can close this issue
